pyproject.toml

```diff
@@ -17,11 +17,12 @@ keywords = [
     'discrete mathematics',
     'math',
 ]
+license = "BSD-3-Clause"
+license-files = ["LICENSE.txt"]
 classifiers = [
     'Development Status :: 5 - Production/Stable',
     'Intended Audience :: Developers',
     'Intended Audience :: Science/Research',
-    'License :: OSI Approved :: BSD License',
```

Comment:
Fwiw we can add [License-expression](https://packaging.python.org/en/latest/specifications/core-metadata/#license-expression) here instead.